lib/Connection.js

```diff
@@ -30,68 +35,95 @@ class Connection {
    * @param {string} db
    * @param {string} user
    * @param {string} pwd
-   * @param {object} [option]
+   * @param {object} [restConfig]
    */
-  constructor(db, user, pwd, option) {
-    this.conn = {};
+  constructor(db, username, password, restConfig) {
+    // setup default values
+    this.conn = {
```

Comment:
`conn` is not really a "connection". I'd suggest `this.options` or `this.conn_options` instead.

Reply:
I agree this name needs to be changed. I left to be `conn` like it was originally but that name is not accurate description as to what it is.

Options is a good name I also was thinking of using config.